### PR TITLE
Add TOC repo step for chairs and TL transition

### PR DESCRIPTION
**Pull request by PushkarJ**
Related issue: https://github.com/cncf/toc/pull/782

### CODE REVIEW

**Comment by TheFoxAtWork** [comment]
TLs are in the TOC repo from what I saw.  Were you going to add the SECURITY.md file updates here as well?

**Review by lumjjb — APPROVED** [review]
LGTM!

File: governance/chair-transition.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ process](chair-proposal-process.md) with specific items that must be completed.
     * [ ] Update codeowners, github settings, README (TOC Liaisons and chairs
       have admin access, tech leads have push access).
     * [ ] Link to official vote email list message in PR descriptions
-  * [ ] [TOC Repo update](https://github.com/cncf/toc/tree/main/tags) - submit PR
+  * [ ] [TOC Repo update](https://github.com/cncf/toc/blob/main/tags/security.md) - submit PR
```
Comment (lumjjb) [inline governance/chair-transition.md:45]:
```suggestion
  * [ ] [TOC Repo update](https://github.com/cncf/toc/blob/main/tags/security.md)
```

File: governance/tech-lead-transition.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ For nomination process please refer to [tech-lead-proposal-process](./tech-lead-
       have admin access, tech leads have push access).
     * [ ] Link to official vote email list message in PR descriptions
     * [ ] Add STAG milestone to track STAG rep for TL
+  * [ ] [TOC Repo update](https://github.com/cncf/toc/blob/main/tags/security.md) - submit PR
```
Comment (lumjjb) [inline governance/tech-lead-transition.md:21]:
```suggestion
  * [ ] [TOC Repo update](https://github.com/cncf/toc/blob/main/tags/security.md) 
```

**Review by lumjjb — COMMENTED** [review]
lint fix remove redundancy

File: governance/tech-lead-transition.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ For nomination process please refer to [tech-lead-proposal-process](./tech-lead-
       have admin access, tech leads have push access).
     * [ ] Link to official vote email list message in PR descriptions
     * [ ] Add STAG milestone to track STAG rep for TL
+  * [ ] [TOC Repo update](https://github.com/cncf/toc/blob/main/tags/security.md)
```
Comment (lumjjb) [inline governance/tech-lead-transition.md:21]:
```suggestion
  * [ ] [TOC Repo update](https://github.com/cncf/toc/blob/main/tags/security.md)
```

**Pull request merged**